src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios'
+
+const marketApi = axios.create({
+    baseURL: 'https://kr-news-portfolio.onrender.com/api'
+})
+
+export const getArticles = ( )=> {
+    return marketApi.get('/articles').then((res) => {
+        return res.data.articles
+    })
+}
```

Comment:
Excellent, glad to see you've extracted out the api functionality to a separate file